askama_derive/src/parser.rs

```diff
@@ -838,20 +838,34 @@ named_args!(block_macro<'a>(s: &'a Syntax<'a>) <Input<'a>, Node<'a>>, do_parse!(
     })
 ));
 
+fn take_raw_block<'a>(
```

Comment:
So not at all clear what you're trying to do here. `block_start` has multiple characters, but you're only checking for the first one, so it doesn't look like this is handling things correctly...

Reply:
Yes, indeed, will try to make it work correctly.